site/content/en/docs/Getting started/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Getting Started"
+linkTitle: "Getting Started"
+weight: 2
+description: >
+  What does your user need to know to try your project?
```

Comment:
@rob8714 whenever you are ready with some content for https://github.com/kubernetes-sigs/kustomize/issues/3973, this would be the place to put it!